src/main/python/envbuilder/cli.py

```diff
@@ -0,0 +1,275 @@
+        bar = ChargingBar('Fetching tags', fill=color('█', fg='green'),
+            max=total, suffix='%(percent).1f%% - %(eta)ds')
+
+        counter = 1
+        for repo in self.repo_map:
+            version = self.api.get_latest(repo, self.args.org)
+            version_key = self.repo_map[repo]
+            self.compose_env_vars[version_key] = f"{version}"
+            bar.next()
+            counter += 1
+        bar.finish()
+
+    def get_client(self):
+        '''Get GitHub API instance'''
+        try:
+            gh_token = os.getenv('GH_TOKEN_PSW')
+
+            try:
+                cabundle = os.getenv('REQUESTS_CA_BUNDLE')
+            except KeyError:
+                cabundle = None
+
+            if cabundle:
+                client = API(bearer_token=gh_token, cabundle=cabundle)
+            else:
+                client = API(bearer_token=gh_token)
+
+            return client
+        except KeyError:
+            print("Please provide [GH_TOKEN_PSW] env var")
+            raise KeyError
+
+    def rate_limit(self):
+        results = self.api.get('/rate_limit')['resources']['core']
+        return results
+
+    def lookup_dependencies(self, deps):
+        updated_versions = {}
+
+        for dep in deps.split(' '):
+            env_file_version_key = f"{dep.upper()}_VERSION"
+            current_version = self.compose_env_vars[env_file_version_key]
+
+            if current_version:
+                is_alpine = 'alpine' in current_version
+
+                lookup = 'alpine' if is_alpine else 'latest'
+                org = 'library'
+
+                # all this if/else is pretty ugly
+                if dep == 'kuiper':
+                    org = 'emqx'
+
+                if dep == 'mosquitto':
+                    image_name = 'eclipse-mosquitto'
+                else:
+                    image_name = dep
+
+                dep_version = self.image_search.get_image_versions(
+                    image_name, org=org, filter=lookup, verbose=False)
+
+                all_versions = dep_version[lookup]
+                # deal with alpine versions a bit different
+
+                version = None  # this should always be the patch semver version
+                if is_alpine:
+                    regex = '^([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)-alpine$'
+
+                    for v in all_versions:
+                        if re.search(regex, v):
+                            version = v
+                            break
+                else:
+                    version = all_versions[0]
+
+                if version is not None:
+                    if current_version != version:
+                        updated_versions[env_file_version_key] = f"{current_version} --> {version}"
+                else:
+                    print(f"Unable to find version for {dep}")
+
+        return updated_versions
+
+    @staticmethod
+    def read_env_file(path='./.env'):
+        with open(path) as f:
```

Comment:
It will be good to have some check around here like
` if os.access(path, os.R_OK):`
What do you think ?

Reply:
Sure. I can add that.